src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -653,31 +604,17 @@ public class ScriptObject
         readonly Action<string> RequestImportSettings;
         readonly Func<bool> RequestSignIn;
         readonly Func<bool> RequestSignOut;
-        readonly Action RequestCloseWindow;
-        readonly Action<bool> RequestCloseWindowPreserve;
 
         /// <summary>
         /// [Obsolete] Constructor for ScriptObject
         /// </summary>
         [Obsolete]
-        public ScriptObject(Action<bool> requestLaunchDynamo, Action<string> requestImportSettings, Func< bool> requestSignIn, Func<bool> requestSignOut, Action requestCloseWindow)
+        public ScriptObject(Action<bool> requestLaunchDynamo, Action<string> requestImportSettings, Func< bool> requestSignIn, Func<bool> requestSignOut)
```

Comment:
This is usually API breaking but I remember in the past we announced that we are not going to guard our UI code to follow backward compatible pattern. @mjkkirschner Correct me if I am wrong